.github/workflows/docs.yml

```diff
@@ -19,12 +19,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout main
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Extract changes for the releases from metainfo.xml
         run: |
           ./scripts/extract-changelog-from-metainfo.py metainfo.xml docs/release-notes.md
 
+      - name: "install dependencies"
+        run: sudo env QTVER=6 SYSDEP_ASSUME_YES=ON ./scripts/install-deps.sh
+      - name: Install-contour
+        run: cmake --preset linux-debug -B build -G Ninja
+      - name: "build"
+        run: cmake --build build
+      - name: "Generate vt documentation"
+        run: ./build/src/contour/contour documentation vt > docs/vt-extensions/index.md
```

Comment:
I think extensions is the wrong place, or is it? It contains most standard ones plus some extensions.... Hmm.....

Reply:
Which place do you prefer?

Reply:
At first I thought internals. But vt sequence documentation targets power users and TUI/CLI devs... So what aber "advanced" user documentation? :thinking: